### DISCUSSION TITLE
Opt out a nested layout from parent layout

### DISCUSSION THREAD

**Discussion by smzm** [vercel/next.js#59844] [Ideas]
### Goals

1. Opt out a specific nested layout from parent layout without using conditional rendering in parent layout.

### Background

![Nested structure route](https://github.com/vercel/next.js/assets/39596095/9daa9d04-cfb9-4466-a4c5-a9705da33eb9)

Next.js app directory sometimes make the structure of project deeply nested and each child need own `layout` file.
But because of **sharing layouts between parent and children**, sometimes it's make serious conflicts between them. 
Also, because of having correct hierarchy route, it's **not meaningful to change the structure** of the project. 

👉 I know there are some solutions like **conditional rendering in parent layouts based on path of route** to extract parent layout and resolve conflict between them, but I think there should be better solution for this. Because sometimes extracting more than one route from parent layout make the code so dirty.

### Proposal

I think setting a specific variable for nested layout to make it **isolated** from parent layouts be a good solution.

**Comment by danieldelcore** [comment]
I'm struggling with this as well, i just want to replace a deeply nested routes layout without completely dismantling my folder structure and duplicating layouts for every other sub-tree.
